### ISSUE TITLE
voms cert password not set

### ISSUE THREAD

**Issue by brucellino** [AAROC/DevOps#340]
The voms server cert password is encrypted and the password is not set in the config.

**Comment by brucellino** [comment]
The error from the voms process (java) is : 

```
2016-10-19 03:44:03.016Z - ERROR [CANLSSLConnectorConfigurator] - SSL initialization error!
java.io.IOException: Can not load the PEM private key: org.bouncycastle.openssl.PEMException: no PasswordFinder specified
```

Seems like we need to add a "password finder"

**Comment by brucellino** [comment]
It seems quite difficult to use the cert with the password - a password reader is required for CANL.. so I think the best workaround is to remove the password from the certificate. I've tested this and it works.

**Comment by brucellino** [comment]
Fixed in acf33496
